src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy

```diff
@@ -49,7 +50,12 @@ class NUnit extends ConventionTask {
     }
 
     File getReportsFolder() {
-        new File(outputFolder, 'reports')
+        if( customReportFolder == null)
```

Comment:
Hi Francois,

I’m not so experienced in developing gradle pluings. Therefore I’m not sure what you mean with using a convention. Maybe you give me a hint what exactly you mean.

Thanks

Regards,
Marco

From: Francois Valdy [mailto:notifications@github.com]
Sent: Thursday, November 05, 2015 3:31 PM
To: Ullink/gradle-nunit-plugin <gradle-nunit-plugin@noreply.github.com>
Cc: Marco Zwyssig <marco@zwyssig.info>
Subject: Re: [gradle-nunit-plugin] support for setting report directory (#10)


In src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy<https://github.com/Ullink/gradle-nunit-plugin/pull/10#discussion_r44016901>:

> @@ -49,7 +50,12 @@ class NUnit extends ConventionTask {

>      }

>

>      File getReportsFolder() {

> -        new File(outputFolder, 'reports')

> +        if( customReportFolder == null)

can you use a convention instead ?

—
Reply to this email directly or view it on GitHub<https://github.com/Ullink/gradle-nunit-plugin/pull/10/files#r44016901>.